modules/applications/pages/download-and-launch.adoc

```diff
@@ -1,4 +1,5 @@
 = Download and launch Documentation
+:page-aliases: ROOT:download-and-launch.adoc
```

Comment:
the :page-aliases: must be here in only 2 scenarios: When the referenced page is moved, or when the referenced page is deleted
```suggestion
```

Reply:
ha yes that's true...